specs/proposals.md

```diff
@@ -47,43 +49,31 @@ output, and the appropriate timestamp, to the [L2 Output Commitment](#l2-output-
 
 ## L2 Output Commitment Construction
 
-This merkle-structure is defined with [SSZ], a type system for merkleization and serialization, used in
-L1 (beacon-chain). However, we replace `sha256` with `keccak256` to save gas costs in the EVM.
-
-[SSZ]: https://github.com/ethereum/consensus-specs/blob/dev/ssz/simple-serialize.md
+The `output_root` is a 32 byte string defined as:
 
```

Comment:
I think this delineate between versions where what is defined below is version 0.

The top level is `keccak256(version || payload)` and in version 0 the `payload` is `state_root || withdrawal_storage_root || latest_block`

Reply:
This allows the version to meaningfully change the definition of the commitment